### ISSUE TITLE
error with discount, qty > 1 and tax

### ISSUE THREAD

**Issue by diegonella** [lukepolo/laracart#265]
Hi, 
I have a next code

```
LaraCart::emptyCart();
LaraCart::destroyCart();

$item = LaraCart::add(1,'Article ID 1', 2, 100, ['tax' => .21], true);

$coupon = new Percentage('10%OFF', 0.10);
LaraCart::addCoupon($coupon);
$coupon->setDiscountOnItem($item);

$itemsRaw = LaraCart::getItems();

$items = [];
foreach( $itemsRaw as $item) {
    

    $items = [
            'id' => $item->id,
            'name' => $item->name,
            'qty' => $item->qty,
            'price' => $item->price(true),
            'discount' => $item->getDiscount(true),
            'netTotal' => $item->netTotal(true),
            'subTotal' => $item->subTotal(true, true, true, false),
    ];
}

$resume = [
    'subTotal' => LaraCart::subTotal(true, false),
    'totalDiscount' => LaraCart::totalDiscount(true),
    'subTotalWithDiscount' => LaraCart::subTotal(true, true),
    'taxTotal' => LaraCart::taxTotal(true),
    'netTotal' => LaraCart::netTotal(true),
    'feeTotals' => LaraCart::feeTotals(true, true),
    'total' => LaraCart::total(true, true),
];

dd($items, $resume, $itemsRaw);

```

output

```
array:7 [
    "id" => 1
    "name" => "Article ID 1"
    "qty" => 2
    "price" => "$100.00"
    "discount" => "$10.00"      // discount by qty, but i have 2
    "netTotal" => "$202.10"     // **I don't understand how he got that value**
    "subTotal" => "$190.00"     // **I think I would have to give 180**
]
```

```
array:7 [
    "subTotal" => "$200.00"
    "totalDiscount" => "$10.00"
    "subTotalWithDiscount" => "$190.00"
    "taxTotal" => "$39.90"
    "netTotal" => "$202.10"
    "feeTotals" => "$0.00"
    "total" => "$229.90"
]
```

```
array:1 [
    "acf3a1f6c711e5a581f86993e3640abe" => LukePOLO\LaraCart\CartItem {
        #itemHash: "acf3a1f6c711e5a581f86993e3640abe"
        #itemModel: null
        #itemModelRelations: []
        +locale: null
        +lineItem: false
        +discount: 0
        +active: true
        +subItems: []
        +couponInfo: []
        +internationalFormat: null
        +options: array:8 [
            "id" => 1
            "qty" => 2
            "name" => "Article ID 1"
            "taxable" => true
            "price" => 100.0
            "tax" => 0.21
            "code" => "10%OFF"
            "coupon" => LukePOLO\LaraCart\Coupons\Percentage {
                +code: "10%OFF"
                +value: 0.1
                +appliedToCart: false
                +options: []
            }
        ]
    }
]
```

the configuration is by default, I have tried different parameters without getting the expected result

**Comment by lukepolo** [comment]
Looks like you found a bug, im writing this test currently, but we did not multiply the qty for the discount.

**Comment by lukepolo** [comment]
https://github.com/lukepolo/laracart/pull/266

**Comment by lukepolo** [comment]
Thank you for the detailed ticket. We have pushed the fix out 1.9.1.
